plexpy/notifiers.py

```diff
@@ -1148,6 +1148,8 @@ def agent_notify(self, subject='', body='', action='', **kwargs):
             attachment = {'title': title
                           }
```

Comment:
Just put the timestamp right inside the `attachment`.

Reply:
ok